src/python/pants/backend/python/typecheck/pyright/rules.py

```diff
@@ -81,26 +142,40 @@ async def pyright_typecheck_partition(
     pex_environment: PexEnvironment,
 ) -> CheckResult:
 
-    root_sources = await Get(
+    root_sources_get = Get(
         SourceFiles,
         SourceFilesRequest(fs.sources for fs in partition.field_sets),
     )
 
     # Grab the inferred and supporting files for the root source files to be typechecked
-    coarsened_sources = await Get(
+    coarsened_sources_get = Get(
         PythonSourceFiles, PythonSourceFilesRequest(partition.root_targets.closure())
     )
 
     # See `requirements_venv_pex` for how this will get wrapped in a `VenvPex`.
-    requirements_pex = await Get(
+    requirements_pex_get = Get(
         Pex,
         RequirementsPexRequest(
             (fs.address for fs in partition.field_sets),
             hardcoded_interpreter_constraints=partition.interpreter_constraints,
         ),
     )
 
-    requirements_venv_pex = await Get(
+    # Look for any/all of the Pyright configuration files (will be modified shortly for `venv` workaround)
```

Comment:
The "will be modified shortly" bit here is referring to the code implemented in this PR right? Maybe say "is modified below"... otherwise it sounds like the feature isn't actually implemented.